toxcore/tox.api.h

```diff
@@ -286,7 +286,10 @@ const FILE_ID_LENGTH              = 32;
  */
 const MAX_FILENAME_LENGTH         = 255;
 
-
+/**
+ * Max legnth of names queried by toxcore
+ */
+const MAX_QUERY_NAME_LENGTH       = 255;
```

Comment:
Just as an additional point, assuming `MAX_QUERY_NAME_LENGTH` is in octets (rather than absolute characters) and UTF-8 is used, certain languages can take up multiple bytes per character (e.g. anything from the CJK character set which take up 3/4 bytes per character). With 64 octets, that limits a purely Chinese name to 21 characters tops (which seems rather short).